src/main/java/com/github/admissionCommittee/dao/DaoFactory.java

```diff
@@ -0,0 +1,91 @@
+package com.github.admissionCommittee.dao;
+
+public class DaoFactory {
+    private DaoFactory() {
+    }
+
+    private static ExamCertificateDao examCertificateDao;
+    private static FacultyDao facultyDao;
+    private static SchoolCertificateDao schoolCertificateDao;
+    private static SheetDao sheetDao;
+    private static SubjectDao subjectDao;
+    private static UserDao userDao;
+    private static volatile DaoFactory daoFactory;
```

Comment:
it usually contains word instance, the same for getter

Reply:
Like this:
...
private static ExamCertificateDaoInstance examCertificateDaoInstance;
public ExamCertificateDaoInstance getExamCertificateDaoInstance() 
...
private static volatile DaoFactory Instance ?

Reply:
I don't really get why you use lazy init of all this instances... but you get the Idea correctly